tests/code/test_fileHandler.py

```diff
@@ -35,6 +35,16 @@ def test_getFileFullPath():
     filePath = handledCourseFile.getFileFullPath(courseFileStorageObject[0].filename)
     assert filePath == 'app/static/files/courseattachments/1/coursefile.pdf'
 
+@pytest.mark.integration
+def test_makingdirectory():
+    #Testing that the file is created and it exists
+    with mainDB.atomic() as transaction:
+        
+        handledEventFile.makeDirectory()
+        handledEventFile.makeDirectory()
```

Comment:
You never actually test that the directory was created. Also, you aren't using the database, so you don't need to use a database transaction.